internal/schedule/penalize/penalize.go

```diff
@@ -38,7 +39,14 @@ func ScheduleCheckForPenalizedNode(node models.Node, repositories repositories.R
 			}
 			if (time.Duration(nodeWithNewCooldown.Cooldown) * time.Minute) > MaxCooldownForPenalizedNode {
 				log.Debugf("Node %s reached maximum cooldown", node.ID)
-				// TODO - remove node
+				err := repositories.NodeRepo.RemoveNodeFromActive(*nodeWithNewCooldown)
+				if err != nil {
+					log.Errorf("Unable to remove node %s from active nodes, because of %v", node.ID, err)
+				}
+				err = whitelist.RemoveNodeFromWhitelisted(node.ID)
```

Comment:
Will this throw error if whitelisting is disabled

Reply:
Good catch, I change `RemoveNodeFromWhitelisted` to not return an error if whitelisting is disabled